src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -12,9 +12,17 @@
   {% set is_logo = logo or theme_logo.get("image_light") or theme_logo.get("image_dark") %}
   {% set image_light = theme_logo.get("image_light") or logo %}
   {% set image_dark = theme_logo.get("image_dark") or logo %}
+
+  {#- Prepare the logo image link based on whether it is local or remote -#}
```

Comment:
I think this whole process can be written in a more compact way: 

```
{% image_ligth = image_light if image_light.startswith("http") else pathto('_static/' + image_light, 1) %}
{% image_dark = image_dark if image_dark.startswith("http") else pathto('_static/' + image_dark, 1) %}
```

I'm guessing that ternary operator are available in jinja2, need to be tested